From b4f6561938e5920d3b72788937242d1fc9d7c448 Mon Sep 17 00:00:00 2001
From: Mark Hindley <mark@hindley.org.uk>
Date: Tue, 5 May 2020 15:01:35 +0100
Subject: [PATCH] Fix for misc/fallocate test build failure.

This was already a known possible failure case. However, in the migration to a
clearer separation of stdout and stderr in the testsuite, the logfile used to
detect the failure was not updated.

Signed-off-by: Mark Hindley <mark@hindley.org.uk>
---
 tests/ts/misc/fallocate | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/ts/misc/fallocate b/tests/ts/misc/fallocate
index 2f77ee928..ec3d2d2f8 100755
--- a/tests/ts/misc/fallocate
+++ b/tests/ts/misc/fallocate
@@ -30,7 +30,7 @@ else
 	# fs type of $TS_OUTDIR, could be used to skip this test early
 	fs_type=$(${TS_CMD_FINDMNT} -n -o FSTYPE -T ${TS_OUTDIR})
 
-	grep -qi "fallocate: fallocate failed:.*not supported" $TS_OUTPUT \
+	grep -qi "fallocate: fallocate failed:.*not supported" $TS_ERRLOG \
 		&& ts_skip "'${fs_type}' not supported"
 fi
 
-- 
2.20.1

